client/blocks/site-address-changer/index.jsx

```diff
@@ -359,7 +359,7 @@ export class SiteAddressChanger extends Component {
 						/>
 						<FormButtonsBar className="site-address-changer__form-footer">
 							<FormButton disabled={ isDisabled } busy={ isBusy } type="submit">
-								{ translate( 'Change Site Address' ) }
+								{ translate( 'Change site address' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Change your site address' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/button-sentence-case)